harness/unittests/test_google_adapter.py

```
@@ -0,0 +1,163 @@
+from __future__ import annotations
+
+import importlib
+import os
+import types as py_types
+import unittest
+import unittest.mock
+from typing import Any, Dict, List
+
+
+class _FakePart:
+    def __init__(self, *, text: str | None = None):
```

Ruff (0.14.3)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)